From 4be81a948a37e71028c45a52dc78489a778b5821 Mon Sep 17 00:00:00 2001
From: Colin Guthrie <colin@mageia.org>
Date: Thu, 26 Apr 2012 23:59:43 +0100
Subject: [PATCH] Make sure to invalidate nscd cache under systemd as needed

---
 sysconfig/network-scripts/network-functions |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sysconfig/network-scripts/network-functions b/sysconfig/network-scripts/network-functions
index 6b5a72d..cb22040 100644
--- a/sysconfig/network-scripts/network-functions
+++ b/sysconfig/network-scripts/network-functions
@@ -466,7 +466,7 @@ change_resolv_conf ()
     if [ $r -eq 0 ]; then
 	[ -x /sbin/restorecon ] && /sbin/restorecon /etc/resolv.conf >/dev/null 2>&1 # reset the correct context
 	/usr/bin/logger -p local7.notice -t "NET" -i "$0 : updated /etc/resolv.conf";
-	[ -e /var/lock/subsys/nscd ] && [ ! -e "$RESOLVCONF_FLAGFILE" ] && /usr/sbin/nscd -i hosts; # invalidate cache
+	[ -n "$(/bin/pidof nscd)" ] && [ ! -e "$RESOLVCONF_FLAGFILE" ] && /usr/sbin/nscd -i hosts; # invalidate cache
     fi;
     return $r;
 }
-- 
1.7.10

