--- initscripts-9.25/sysconfig/network-scripts/network-functions.nm_check~	2012-06-06 20:54:46.077685906 +0200
+++ initscripts-9.25/sysconfig/network-scripts/network-functions	2012-06-06 20:34:09.187149165 +0200
@@ -147,7 +147,7 @@ source_config ()
     else
 	ISALIAS=no
     fi
-    is_true $NM_CONTROLLED && USE_NM=true
+    is_true $NM_CONTROLLED && test `pidof NetworkManager` && USE_NM=true
 }
 
 
