--- initscripts-9.44/rc.d/init.d/netconsole.orig	2013-07-31 16:30:29.481047071 +0100
+++ initscripts-9.44/rc.d/init.d/netconsole	2013-07-31 16:32:13.345041002 +0100
@@ -63,7 +63,7 @@
 		target=$host
 	fi
 	if [ -z "$SYSLOGMACADDR" ]; then
-		arp=$(LANG=C /sbin/arping -f -c 1 -I $DEV $target 2>/dev/null | awk '/ reply from .*[.*]/ { print gensub(".* reply from .* \\[(.*)\\].*","\\1","G"); exit }')
+		arp=$(LANG=C /usr/sbin/arping -f -c 1 -I $DEV $target 2>/dev/null | awk '/ reply from .*[.*]/ { print gensub(".* reply from .* \\[(.*)\\].*","\\1","G"); exit }')
 		[ -n "$arp" ] && echo "SYSLOGMACADDR=$arp"
 	fi	
 }	
--- initscripts-9.44/sysconfig/network-scripts/ifup-eth.orig	2013-07-31 16:30:33.855046815 +0100
+++ initscripts-9.44/sysconfig/network-scripts/ifup-eth	2013-07-31 16:36:15.431026858 +0100
@@ -423,7 +423,7 @@
 
         if ! LC_ALL=C ip addr ls ${REALDEVICE} | LC_ALL=C grep -q "${ipaddr[$idx]}/${prefix[$idx]}" ; then
             [ "${REALDEVICE}" != "lo" ] && [ "${arpcheck[$idx]}" != "no" ] && \
-            if ! /sbin/arping -q -c 2 -w 3 -D -I ${REALDEVICE} ${ipaddr[$idx]} ; then
+            if ! /usr/sbin/arping -q -c 2 -w 3 -D -I ${REALDEVICE} ${ipaddr[$idx]} ; then
                 net_log $"Error, some other host already uses address ${ipaddr[$idx]}."
                 mdv-network-event connection_failure ${DEVICE}
                 exit 1
@@ -441,9 +441,9 @@
 
         # update ARP cache of neighboring computers
         if [ "${REALDEVICE}" != "lo" ]; then
-          /sbin/arping -q -A -c 1 -I ${REALDEVICE} ${ipaddr[$idx]}
+          /usr/sbin/arping -q -A -c 1 -I ${REALDEVICE} ${ipaddr[$idx]}
           ( sleep 2;
-            /sbin/arping -q -U -c 1 -I ${REALDEVICE} ${ipaddr[$idx]} ) > /dev/null 2>&1 < /dev/null &
+            /usr/sbin/arping -q -U -c 1 -I ${REALDEVICE} ${ipaddr[$idx]} ) > /dev/null 2>&1 < /dev/null &
         fi
     done
     fi
--- initscripts-9.44/sysconfig/network-scripts/ifup-aliases.orig	2013-07-31 16:44:14.877998845 +0100
+++ initscripts-9.44/sysconfig/network-scripts/ifup-aliases	2013-07-31 16:44:23.194998359 +0100
@@ -260,7 +260,7 @@
                [ "${parent_device}" != "lo" ] &&  [ "${ARPCHECK}" != "no" ] && \
                is_available ${parent_device} && \
                grep -qswi "up" /sys/class/net/${parent_device}/operstate && \
-               if ! /sbin/arping -q -c 2 -w 3 -D -I ${parent_device} ${IPADDR} ; then
+               if ! /usr/sbin/arping -q -c 2 -w 3 -D -I ${parent_device} ${IPADDR} ; then
                    net_log $"Error, some other host already uses address ${IPADDR}."
                    return 1
                fi
