diff -Naur initscripts-10.01/Makefile initscripts-10.01.tpg/Makefile
--- initscripts-10.01/Makefile	2018-08-06 11:06:05.000000000 +0000
+++ initscripts-10.01.tpg/Makefile	2018-10-02 20:43:21.832843141 +0000
@@ -18,18 +18,18 @@
 
 # Normally /usr/local is used. However, it does not make sense for us to use it
 # here, as it just complicates things even further.
-prefix         = /usr
-exec_prefix    = $(prefix)
-bindir         = $(prefix)/bin
-sbindir        = $(prefix)/sbin
-libdir         = $(prefix)/lib
-libexecdir     = $(exec_prefix)/libexec
-datarootdir    = $(prefix)/share
-datadir        = $(datarootdir)
-mandir         = $(datadir)/man
-sysconfdir     = /etc
-localstatedir  = /var
-sharedstatedir = $(localstatedir)/lib
+prefix         ?= /usr
+exec_prefix    ?= $(prefix)
+bindir         ?= $(prefix)/bin
+sbindir        ?= $(prefix)/sbin
+libdir         ?= $(prefix)/lib
+libexecdir     ?= $(exec_prefix)/libexec
+datarootdir    ?= $(prefix)/share
+datadir        ?= $(datarootdir)
+mandir         ?= $(datadir)/man
+sysconfdir     ?= /etc
+localstatedir  ?= /var
+sharedstatedir ?= $(localstatedir)/lib
 
 VERSION       := $(shell gawk '/Version:/ { print $$2 }' initscripts.spec)
 NEXT_VERSION  := $(shell gawk '/Version:/ { print $$2 + 0.01}' initscripts.spec)
@@ -68,7 +68,11 @@
 
 install-usr:
 	install -m 0755 -d $(DESTDIR)$(prefix)
-	cp -a        usr/* $(DESTDIR)$(prefix)/
+	install -m 0755 -d $(DESTDIR)/lib
+	install -m 0755 -d $(DESTDIR)/sbin
+	cp -a        usr/libexec $(DESTDIR)$(prefix)/
+	cp -a        usr/lib/* $(DESTDIR)/lib
+	cp -a        usr/sbin/* $(DESTDIR)/sbin
 
 install-network-scripts: install-usr install-etc
 	install -m 0755 -d      $(DESTDIR)$(sysconfdir)/sysconfig/network-scripts
diff -Naur initscripts-10.01/src/Makefile initscripts-10.01.tpg/src/Makefile
--- initscripts-10.01/src/Makefile	2018-08-06 11:06:05.000000000 +0000
+++ initscripts-10.01.tpg/src/Makefile	2018-10-02 20:46:40.386840395 +0000
@@ -12,12 +12,12 @@
 
 # Normally /usr/local is used. However, it does not make sense for us to use it
 # here, as it just complicates things even further.
-prefix      = /usr
-bindir      = $(prefix)/bin
-sbindir     = $(prefix)/sbin
-libdir      = $(prefix)/lib
-
-CC          = gcc
+prefix      ?= /usr
+bindir      ?= $(prefix)/bin
+sbindir     ?= $(prefix)/sbin
+libdir      ?= $(prefix)/lib
+udevdir     ?= $(libdir)/udev
+CC         ?= gcc
 CFLAGS     += $(RPM_OPT_FLAGS) -Wall -D_GNU_SOURCE -fPIE
 LDFLAGS    += $(RPM_LD_FLAGS) -pie -z relro -z now
 
@@ -28,12 +28,12 @@
 install: all
 	install -m 0755 -d $(DESTDIR)$(bindir)
 	install -m 0755 -d $(DESTDIR)$(sbindir)
-	install -m 0755 -d $(DESTDIR)$(libdir)/udev
+	install -m 0755 -d $(DESTDIR)$(udevdir)
 	install -m 0755 build/usleep         $(DESTDIR)$(bindir)
 	install -m 0755 build/consoletype    $(DESTDIR)$(sbindir)
 	install -m 0755 build/genhostid      $(DESTDIR)$(sbindir)
 	install -m 0755 build/usernetctl     $(DESTDIR)$(sbindir)
-	install -m 0755 build/rename_device  $(DESTDIR)$(libdir)/udev
+	install -m 0755 build/rename_device  $(DESTDIR)$(udevdir)
 
 clean:
 	rm -f build/*
