--- initscripts-9.34/mandriva/dm.hald~	2012-09-05 19:13:54.567903130 +0200
+++ initscripts-9.34/mandriva/dm	2012-09-05 19:14:14.449173585 +0200
@@ -13,7 +13,8 @@
 #
 ### BEGIN INIT INFO
 # Provides: dm
-# Should-Start: network-auth acpid lircmd xfs haldaemon
+# Required-Start: messagebus
+# Should-Start: network-auth acpid lircmd xfs
 # Default-Start: 5 7
 # Short-Description: Launches the graphical display manager
 # Description: This startup script launches the graphical display manager.
